src/Components/EnvironmentPicker/EnvironmentPickerManager.ts

```diff
@@ -0,0 +1,374 @@
+import { AzureResourceTypes, IAzureResource } from '../../Models/Constants';
+import {
+    ContainersLocalStorageKey,
+    EnvironmentsLocalStorageKey,
+    SelectedContainerLocalStorageKey,
+    SelectedEnvironmentLocalStorageKey,
+    StorageAccountsLocalStorageKey
+} from '../../Models/Constants/Constants';
+import {
+    ADTEnvironmentInLocalStorage,
+    ADTSelectedEnvironmentInLocalStorage,
+    StorageAccountsInLocalStorage,
+    StorageAccountToContainersMapping
+} from './EnvironmentPicker.types';
+
+class EnvironmentPickerManager {
```

Comment:
can we export functions independently like we do for `Utils.ts`? That way we can import individual ones and not always have to prefix them with `EnvironmentPickerManager` which is quite verbose and makes it harder to read.